autosklearn/estimators.py

```diff
@@ -28,7 +28,8 @@ def __init__(self,
                  shared_mode=False,
                  disable_evaluator_output=False,
                  get_smac_object_callback=None,
-                 smac_scenario_args=None):
+                 smac_scenario_args=None,
+                 log_config=None):
```

Comment:
Please add a trailing comma, rename the argument and move the closing bracket to a new line.